.typos.toml

```diff
@@ -0,0 +1,22 @@
+# this is the configuration file of typos (spell checker)
+# https://github.com/crate-ci/typos
+
+[files]
+# excluded file
+extend-exclude = [
+	"go.sum", "go.mod", # go.sum and go.mod contains URLs with hash, they can provide false positive
```

Comment:
hashes are all same length so we could ignore them with regex, without ignoring the files.

Reply:
in fact, there is no problem with go.mod and go.sum, so I removed the exceptions.